[Implementation]-[Create test-cases and deploy Allure report]

JCamiloMRincon opened the pull request:
This pull-request contains all the implementation done during the technical challenge:

- Create Playwright UI and API test-cases
- Generate Allure reports
- Create YAML file with the workflow for deploying Allure reports
- Host the report in Github Pages

JCamiloMRincon committed:
Changes

JCamiloMRincon committed:
Update main.yml

JCamiloMRincon committed:
Update main.yml

JCamiloMRincon committed:
Changes

JCamiloMRincon committed:
Merge branch 'cmortigo/test1' of https://github.com/JCamiloMRincon/on-boarding-tasks into cmortigo/test1

JCamiloMRincon committed:
Create new branch with Node

JCamiloMRincon committed:
Add Setup Fixture class

JCamiloMRincon committed:
Generate Allure Results

JCamiloMRincon committed:
Update main.yml

JCamiloMRincon committed:
Use npx allure-commandline

JCamiloMRincon committed:
Update main.yml

JCamiloMRincon committed:
Update main.yml

JCamiloMRincon committed:
Update main.yml

JCamiloMRincon committed:
Update main.yml

JCamiloMRincon committed:
Complete technical challenge

JCamiloMRincon committed:
Merge branch 'cmortigo/test3' of https://github.com/JCamiloMRincon/on-boarding-tasks into cmortigo/test3

JCamiloMRincon committed:
Add .gitignore file

Dmytro-Hazashvili-EPAM commented:
1. Add .gitignore file and clean your PR from system files
2. Follow a structure of Core project, Tests project.

JCamiloMRincon committed:
Delete unneccesary files and folders and modify again the .gitignore file

JCamiloMRincon committed:
Modify project configurations

JCamiloMRincon commented:
@Dmytro-Hazashvili-EPAM I have solved the two issues in the PR

![image](https://github.com/user-attachments/assets/42cfc4c7-2277-46e4-9b4b-56fc3c759a38)

- I cleaned the solution, where the files and folders included in the `.gitignore` file were removed and then the commit was created. With that, we ensure that our solution will be cleaner.
- I modified the name of the folder where the core project is included. Now the folder structure of the solution includes core folder, test folder and workflows folder.

JCamiloMRincon committed:
Solve merge conflicts
